### ISSUE TITLE
Feature Request: Automatically change theme based on system theme

### ISSUE THREAD

**Issue by Seo-Rii** [mallowigi/material-theme-issues#36]
**Is your feature request related to a problem? Please describe.**
I usually use a laptop to work, which means that my working environment changes frequently. This means that the surrounding environment, including illumination, changes frequently, so I often switch to dark mode and light mode.
Most programs follow the settings well if they change the system theme, but Jetbrain's IDE does not. I hope there is a function that detects the transition of the system theme and automatically changes the theme.

**Describe the solution you'd like**
I wish there was a kind of theme palette. It would be nice to designate the theme to be used in the light theme and the dark theme, respectively, and automatically change it according to the system theme.

**Comment by mallowigi** [comment]
There is a feature in JetBrains settings that allows you to set your preferred light and dark themes respectively. It should be in the Appearance settings if I remember correctly.

**Comment by Seo-Rii** [comment]
I didn't know it had this function. Thank you for letting me know!

**Comment by shearn89** [comment]
For anyone who googles this in future - my theme seemed to be stuck on dark mode, but the switch was working. I set Appearance -> Theme to "Sync with OS", then ticked my preferred light/dark themes in the dropdown. Then I set the Material plugin to use the "light" theme I'd chosen (as it was daytime). Lastly, going to the appearance settings and toggling from light to dark mode changed the theme correctly.
